setup/steps/apply_monitoring_user_password.go

```diff
@@ -0,0 +1,86 @@
+package steps
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/lib/pq"
+	"github.com/pganalyze/collector/config"
+	s "github.com/pganalyze/collector/setup/state"
+	mainUtil "github.com/pganalyze/collector/util"
+)
+
+var ApplyMonitoringUserPasswd = &s.Step{
+	Description: "Apply monitoring user password",
```

Comment:
Would be good to clarify whether that this is modifying the pganalyze collector config (not the database)